lib/zipkin-tracer/config.rb

```diff
@@ -62,6 +66,15 @@ def adapter
       sampled_as_boolean: true
     }
 
+    def parse_record_on_server_receive(tag_names)
```

Comment:
I don't know what's idiomatic, but seems like it would be easier to see this if this function were in the rack class (ex tag filter was passed to it vs moving the logic here). For example, it would make it easier to see if you change "parse_record_on_server_receive" to "parse_tags" and no meaning is loss. I was looking to see if any tags were added on server send and needed to scroll to the rack handler file to find that out..